.pre-commit-config.yaml

```
@@ -50,11 +50,11 @@ repos:
     hooks:
       - id: mypy
         # empty args needed in order to match mypy cli behavior
-        args: []
+        args: ['--no-incremental']
         entry: mypy src/
         pass_filenames: false
         additional_dependencies:
-          - ansible-lint>=5.0.10
+          - ansible-compat>=0.2.0
```

I don't think we should depend on non-stable releases, and once there is a stable release, we should use semver to make sure there are no breaking changes.
```suggestion
          - ansible-compat>=1.0.0,<2.0.0
```